packages/mux-player/src/index.ts

```diff
@@ -80,7 +80,7 @@ const PlayerAttributes = {
 };
 
 function getProps(el: MuxPlayerElement, state?: any): MuxTemplateProps {
-  return {
+  const props = {
```

Comment:
NOTE: Intentionally committing this, since I frequently have to refactor to add e.g. console.logs or similar in this function.

Reply:
I've done that a bunch as well, so, 👍🏻